test/steps/http.go

```diff
@@ -26,6 +26,7 @@ import (
 func registerHTTPSteps(s *godog.Suite, data *Data) {
 	s.Step(`^HTTP GET request on service "([^"]*)" with path "([^"]*)" is successful within (\d+) minutes$`, data.httpGetRequestOnServiceWithPathIsSuccessfulWithinMinutes)
 	s.Step(`^HTTP GET request on service "([^"]*)" with path "([^"]*)" should return an array of size (\d+) within (\d+) minutes$`, data.httpGetRequestOnServiceWithPathShouldReturnAnArrayofSizeWithinMinutes)
+	s.Step(`^HTTP GET request on service "([^"]*)" with path "([^"]*)" is stored in variable "([^"]*)"$`, data.httpGetRequestOnServiceWithPathAndSetVariable)
```

Comment:
Same for this step.

Reply:
I didn't want to remove them as we might need them in the future. 
But I guess it makes sense what you say, we have the git history anyway.